elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -237,6 +238,20 @@ void ElevationMapping::visibilityCleanupThread()
 void ElevationMapping::pointCloudCallback(
     const sensor_msgs::PointCloud2& rawPointCloud)
 {
+  // Check if point cloud has corresponding robot pose at the beginning
+  if(not receivedFirstMatchingPointcloudAndPose_) {
+    const double oldestPoseTime = robotPoseCache_.getOldestTime().toSec();
+    const double currentPclTime = rawPointCloud.header.stamp.toSec();
```

Comment:
Given that the package includes full words in variable names, to have more descriptive ones. (e.g. pointCloud in rawPointCloud). I would rename this variable as currentPointCloudTime.